portfolio/src/main/webapp/script.js

```diff
@@ -142,41 +147,65 @@ function getMessage() {
 function handleResponse(response) {
   const textPromise = response.text();
   
-  textPromise.then(addSingleMessageToDom);
+  textPromise.then(addSingleCommentToDom);
 }
 
 /**
- * Adds a single message {@code message} inside of the div
+ * Adds a single {@code comment} inside of the div
  * message-container. 
  */
-function addSingleMessageToDom(message) {
-  const messageContainer = document.getElementById('message-container');
-  messageContainer.innerHTML = message;
+function addSingleCommentToDom(comment) {
+  const commentContainer = document.getElementById(COMMENT_CONTAINER);
+  commentContainer.innerHTML = message;
 }
 
 /**
- * Adds {@code messages} to the DOM as list elements.
+ * Adds {@code comments} to the DOM as list elements.
  */
-function addMultipleMessagesToDom(messages) {
-  const messageContainer = document.getElementById('message-container');
-  const ulElement = document.createElement('ul');
-  messageContainer.appendChild(ulElement);
+function addMultipleMessagesToDom(comments) {
+  const commentContainer = document.getElementById(COMMENT_CONTAINER);
 
-  for (let key in messages) {
-    appendTextToList(messages[key], ulElement);
+  // Removes the ul tag in the container if there is one to prevent having
+  // multiple sets of ul tags every time the number of comments is changed.
+  if (commentContainer.firstChild) {
+    commentContainer.removeChild(commentContainer.firstChild);
   }
+  const ulElement = document.createElement('ul');
+  commentContainer.appendChild(ulElement);
+
+  comments.forEach((comment) => {
+    appendTextToList(comment.text, ulElement);
+  });
 }
 
 /**
- * Fetches the message from the JSON server /data and adds it to the DOM.
- *
- * <p>Method is called everytime the page is refreshed.
+ * Fetches the comment from the JSON server /data and adds it to the DOM.
+ * @param pageReloadBoolean Indicates whether the method is called
+ * when the page refreshes or when a new comment amount is inputted.
```

Comment:
The JSDoc should say something like:

If {@code pageReloadBoolean} is true, then fetches the comment from the JSON server /data and add its to the DOM. Otherwise, ...

This way you do not need @param.

Reply:
Understood!
Done!